lib/features.js

```diff
@@ -0,0 +1,67 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+
+module.exports = config => {
+  const lastAccessTimeUpdates = config.lastAccessTimeUpdates
+  const signinConfirmation = config.signinConfirmation
+
+  return {
+    isSampledUser: isSampledUser,
+
+    isLastAccessTimeEnabled (uid, email) {
+      return lastAccessTimeUpdates.enabled && (
+        isSampledUser(uid, lastAccessTimeUpdates.sampleRate) ||
+        new RegExp(lastAccessTimeUpdates.enabledEmailAddresses).test(email)
+      )
+    },
+
+    isSigninConfirmationEnabled (uid, email, request) {
```

Comment:
There were test coverage for it, https://github.com/mozilla/fxa-auth-server/blob/master/test/local/account_routes.js#L916 but not very obvious :/. I like what you have done extracting out the logic, makes finding the tests easier.